### PR TITLE
Create projects and project_feedbacks tables

### LINKED ISSUE #14: Create projects table/model
- [x] student_id
- [x] name
- [x] mod
- [x] program
- [x] instructor_comments
- [x] student_comments

### LINKED ISSUE #15: Create project_feedback table/model
- [x] instructor_id (fk to users, id)
- [x] project_id (fk to projects, id)
- [x] project_rubric_id (fk to module_rubrics, id)
- [x] score
- [x] comment

### PR DESCRIPTION
**Pull request by Gvieve**
### Add
* projects table, model, spec, validations, factory and associations
* projects table, model, spec, validations, factory and associations
### Update / Remove
* users spec and model to add new association to projects and project_feedbacks

Closes #14
closes #15

### CODE REVIEW

File: db/migrate/20210514221715_create_projects.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :projects do |t|
+      t.string :name
+      t.string :mod
+      t.string :program
+      t.string :instructor_comments
+      t.string :student_comments
+      t.references :student, references: :users, foreign_key: {to_table: :users}
```
Comment (b-enji-cmd) [inline db/migrate/20210514221715_create_projects.rb:9]: Fancy!

**Review by b-enji-cmd — APPROVED** [review]
LGTM!!

**Comment by jequeza** [comment]
Looks great dev! Awesome work!

**Review by jequeza — APPROVED** [review]
Ship it!

**Pull request merged**